src/mmw/js/src/modeling/gwlfe/quality/templates/table.html

```diff
@@ -44,3 +44,16 @@
 <div class="downloadcsv-link" data-action="download-csv-granular">
     <i class="fa fa-download"></i> Download this data
 </div>
+<div class="download-bmp row">
+    <hr />
+    <div class="col-sm-2 text-center">
+        <i class="fa fa-lightbulb-o"></i>
+    </div>
+    <div class="col-sm-10">
+        <p>
+            Get our <a href="https://raw.githubusercontent.com/WikiWatershed/MMW-BMP-spreadsheet-tool/master/MMW_BMP_Spreadsheet_Tool.xlsx" target="_blank" rel="noreferrer noopener">BMP spreadsheet tool</a> for advanced simulation of urban BMPs.
```

Comment:
I can't find any decent documentation, but I thought linking to `raw.githubusercontent.com/path/to/file` was discouraged, but maybe not a big deal.

Reply:
I'll replace with https://github.com/WikiWatershed/MMW-BMP-spreadsheet-tool/raw/master/MMW_BMP_Spreadsheet_Tool.xlsx which seems to redirect to that, but is probably more permanent.